Cargo.toml

```diff
@@ -1,5 +1,7 @@
 [profile.release]
 panic = "unwind"
+opt-level = 3
+lto = true
```

Comment:
This makes it much slower to compile. Perhaps we should create a `production` profile only for this 🤔

Reply:
Yes, your call here, I can create it, if you want.

Reply:
Better that way, yes. We can only use the `production` profile in the CI when building the runtime with `srtool` and this way we as developers do not have to pay the penalty when, for instance, testing with zombienet.

Reply:
The node should also be built with `lto` when they get released.

Reply:
Yes, in that case the `production` profile should also be added to the docker image upload job.